### PR TITLE
Add me ...

### PR DESCRIPTION
**Pull request by endymion1818**
Would you like to add me as a friend?

https://open.spotify.com/track/4DnmtCxY0BkWrX2DSxRvJP?si=31513a3fc3cd4002

### CODE REVIEW

File: data/members.json [hunk 1]
```diff
@@ -7,6 +7,8 @@
   { "name": "Max Böck", "feed": "https://mxb.dev/feed.xml", "url": "https://mxb.dev/" },
   { "name": "Matthew Brandt", "feed": null, "url": "https://mattytwo.shoes/" },
   { "name": "Scott Spence", "feed": "https://scottspence.com/rss.xml", "url": "https://scottspence.com" },
+  { "name": "Scott Spence", "feed": "https://scottspence.com/rss.xml", "url": "https://scottspence.com" },
```
[the remark this answers is not in the document]
Comment (endymion1818) [reply data/members.json:10]: oh drat, thanks

**Comment by whitep4nth3r** [comment]
I'm so sorry you are at the receiving end of so many merge conflicts 😅

**Comment by whitep4nth3r** [comment]
I want to merge yours before I merge others because if I merge others, you'll have more conflicts!

**Comment by endymion1818** [comment]
Ha ha appreciate it Salma thanks

**Pull request merged**